.github/workflows/customer-service.yml

```diff
@@ -0,0 +1,27 @@
+name: customer-service-pipeline
+on:
+  push:
+    paths:
+      - 'customer-service/**'
+    branches: [ develop ]
+  pull_request:
+    paths:
+      - 'customer-service/**'
```

Comment:
It should only be triggered if git detects changes in this path. 
see also here https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestpaths